server/connector/data_sink.hpp

```diff
@@ -188,8 +188,9 @@ class RocksDBDataSink : public velox::connector::DataSink {
   std::vector<catalog::Column::Id> _column_ids;
   velox::memory::MemoryPool& _memory_pool;
   SliceVector _row_slices;
-  primary_key::Keys _row_keys;
+  primary_key::Keys _cell_keys_buffers;  // buffer per row
```

Comment:
NIT: I don't like name. it's meaningless, maybe just `_keys`? It's lock (row) keys and rocksdb (column) keys